windows/Accelerometer.cs

```diff
@@ -57,22 +53,18 @@ public void startUpdates()
             {
                 _accelerometer = Sensors.Accelerometer.GetDefault();
                 if (_accelerometer == null) throw new Exception("No Accelerometer found");
+
+                this.setUpdateInterval(this.interval);
             }
             _accelerometer.ReadingChanged += new TypedEventHandler<Sensors.Accelerometer, Sensors.AccelerometerReadingChangedEventArgs>(ReadingChanged);
         }
 
         [ReactMethod]
         public void stopUpdates()
         {
-            //throw new NotImplementedException();
+            _accelerometer.ReadingChanged -= new TypedEventHandler<Sensors.Accelerometer, Sensors.AccelerometerReadingChangedEventArgs>(ReadingChanged);
```

Comment:
I asked myself the same question at the beginning too...

Check the remarks on that page : https://msdn.microsoft.com/en-us/library/system.delegate.op_equality.aspx

> Two delegates of the same type with the same targets, methods, and invocation lists are considered equal.